### PR TITLE
Remove unused layer from survey detail map

### PR DESCRIPTION
**Pull request by maurizi**

### CODE REVIEW

File: src/nyc_trees/apps/survey/views.py [hunk 1]
```diff
@@ -415,16 +415,17 @@ def _get_map_another_popup_context(request):
...
         cursor.execute(_SURVEY_DETAIL_QUERY, [survey_id])
         trees = [tree[0] for tree in cursor]
+
     ctx = {
         'survey_id': survey_id,
         'blockface_id': survey.blockface_id,
         'trees': json.dumps(trees),
-        'layer': get_context_for_reservations_layer(request),
-        'bounds': _user_reservation_bounds(request.user),
+        'bounds': list(survey.blockface.extent),
```
Comment (steventlamb) [inline src/nyc_trees/apps/survey/views.py]: getting:
screenshot from 2015-04-06 11 29 48

Comment (maurizi) [inline src/nyc_trees/apps/survey/views.py]: Ah! typo, will fix!

**Comment by maurizi** [comment]
@steventlamb sorry about the wait, typo is fixed now

**Comment by steventlamb** [comment]
+1

**Pull request merged**